src/views/auth/AuthRoute.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route, Redirect } from 'react-router-dom';
+
+const AuthService = {
```

Comment:
What are you trying to achieve with this authService

Reply:
It is part of the configuration for checking if a route is authenticated or not. If a user is not logged in and tries visit a route that is authenticated it redirects him to the login route.  If you view the login component you will see its usage. Once a person successfully logs in they are authenticated and can view all protected routes.

Reply:
And as it stands now, it is a dummy representation of the App State. The ticket required that we have a dummy login route setup. After the Redux setup has been merged,  the person assigned the login feature will have to upgrade it accordingly.

Reply:
alright